teachertool/src/components/styling/RubricWorkspace.module.scss

```diff
@@ -0,0 +1,17 @@
+.panel {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+    background-color: var(--pxt-page-background);
+    color: var(--pxt-page-foreground);
+
+    div[class*="tt-tabgroup"] {

```

Comment:
nit (and arguably not really an issue at all): is the tt prefix necessary with the module scoping this to just the RubricWorkspace anyway?

Reply:
This is a case where the parent control wants to modify a child component's style. It is selecting the child element by class name, so we're not using a module css-generated name.